tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferences/names.tsx

```diff
@@ -154,7 +154,7 @@ export function NameInput(props: NameInputProps) {
             <Input
               autoSelect
               expensive
```

Comment:
As per discord, got added after `onChange` got changed from "blur and enter" to "any input"- but probably doesn't make much sense to be here when we only do stuff on `onBlur` and `onEscape`, given as per the docs it only applies to `onChange` in the first place